playground-common/src/Debug/Trace/Extra.js

```diff
@@ -0,0 +1,14 @@
+/*eslint-env node*/
+'use strict';
+
+exports.traceTime = function () {
+    return function (x) {
+        return function (k) {
+            var t0 = performance.now();
+            var res = k();
+            var t1 = performance.now();
+            console.log(x + (t1 - t0) + " milliseconds.");
+            return res;
+        };
+    };;
```

Comment:
Double semicolon.

Reply:
How about:

```js
console.time(x);
var res = k();
console.timeEnd(x);
return res;
```

https://developer.mozilla.org/en-US/docs/Web/API/console#Timers

Reply:
Also, how about we call `x` "label" and `k` "action"?

Reply:
nice, changed